tox.ini

```diff
@@ -37,6 +37,11 @@ deps=flake8
 basepython=python3
 commands=flake8 model_bakery {posargs}
 
+[testenv:isort]
+deps=isort
+basepython=python3
+commands=isort model_bakery {posargs}
```

Comment:
Can we add the [`--check-only`](https://github.com/timothycrosley/isort#the---check-only-option) flag? Without this flag, the CI only updates the import order and then `black` will execute on different files from what the contributor committed.

I've removed `isort` because I was losing a lot of time just trying to figure out what was happening. So, if we only run the check, the CI job will raise more detailed error messages so people can update their codes.

Reply:
Good point! I've added it in a983c3d